rdmd.d

```diff
@@ -902,12 +882,11 @@ string which(string path)
         foreach (extension; extensions)
         {
             string absPath = buildPath(envPath, path ~ extension);
-            yap("stat ", absPath);
-            if (exists(absPath) && isFile(absPath))
+            if (existsAsFile(absPath))
```

Comment:
Just use `if (exists(absPath) && isFile(absPath))` here.  It's simpler, it saves you defining an unnecessary extra wrapper function, and it makes the verbose report more correct.